package.json

```diff
@@ -55,5 +55,6 @@
     },
     "npm": false
   },
-  "version": "1.2.0"
+  "version": "1.2.0",
+  "packageManager": "pnpm@9.12.3+sha512.cce0f9de9c5a7c95bef944169cc5dfe8741abfb145078c0d508b868056848a87c81e626246cb60967cbd7fd29a6c062ef73ff840d96b3c86c40ac92cf4a813ee"
```

Comment:
was this put here intentionally?

Reply:
upps... yes... forget to move to npm v18... thanks for hint

Reply:
when everything works fine we will move everything in next time to node 20 & pnpm 9